models/fast_fourier_transform/modelparameters.m

```diff
@@ -13,10 +13,10 @@
 
 %% Model parameters 
 mp.FFT_size = 128;
-% XXX: an unsigned int of mp.FFT_size_Nbits only goes up to mp.FFT_size - 1; not sure if that's intended or a bug 
+% An unsigned int of mp.FFT_size_Nbits only goes up to mp.FFT_size - 1; not sure if that's intended or a bug 
 mp.FFT_size_Nbits = log2(mp.FFT_size);
 mp.FFT_size_half = mp.FFT_size/2;
-mp.FFT_frame_shift = mp.FFT_size/4;  % XXX: Changiing this from a divide by four implies substantial architectural changes
```

Comment:
@fe-tdavis I wouldn't call this a typo. `XXX` is a code tag like `TODO` or `NOTE`, often taken as a warning about potential pitfalls, etc. `XXX` usage is pretty vague across the board, though... Either way, removing the code tag is fine.

Reply:
@tvannoy True, but "Changiing" is the typo.  I don't like the `XXX` because is pretty vague. If a tag is going to be used, it should be more specific.

Reply:
Agreed about using specific tags. I should be better about that. `WARNING` would be more appropriate. I didn't even notice the actual typo haha. You're very good at catching all of my typos.

Reply:
haha, I've had a lot of practice between the dissertation and writing for fun.  Just wait until you have to go through your dissertation ;)